modin/_compat/pandas_api/latest/window.py

```diff
@@ -79,3 +79,25 @@ def __init__(
             ],
             axis,
         )
+
+@append_to_docstring("Compatibility layer for 'latest pandas' for Expanding.")
+@_inherit_docstrings(pandas.core.window.expanding.Expanding)
+class LatestCompatibleExpanding(BaseCompatibleExpanding):
+    def __init__(
+        self,
+        dataframe,
+        min_periods=1,
+        center=None,
```

Comment:
default should be `False` right?

Reply:
Shouldn't it be same as what Pandas expanding API takes as defaults? That's how it's done for Rolling, so it would be the same for Expanding. Wouldn't it?

Reply:
@mdatre sorry, default for center should be `False` right? See here: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.expanding.html. I don't think it's a big deal either way.

Reply:
Oh I see. I borrowed "None" from the top line of Pandas. Will fix this.

Reply:
@pyrito If you see the default expanding API here https://github.com/devin-petersohn/modin/blob/service/init-prod/modin/_compat/pandas_api/latest/base.py#L127 it has center=None. This init is going to have that borrowed from this upper DF API. So I think we should leave it as "None"?!

Reply:
The [documentation](https://pandas.pydata.org/pandas-docs/version/1.4/reference/api/pandas.DataFrame.expanding.html) for pandas 1.4 says `center=None` in the signature, but there is `center: bool, default False` in the parameter description, but the implementation uses `None`. So we want `None`.